torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2263,8 +2263,31 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad, **kwargs)
 # TODO: clamp shares tensors among its sample inputs --- we should prohibit this!
 def sample_inputs_clamp(op_info, device, dtype, requires_grad, **kwargs):
     tensors = (
-        make_tensor((2, 3, 2), device=device, dtype=dtype, low=None, high=None, requires_grad=requires_grad),
-        make_tensor((2, 0, 3), device=device, dtype=dtype, low=None, high=None, requires_grad=requires_grad),
+        make_tensor((S, M, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
```

Comment:
Add a broadcasting case here, too

Reply:
There are already boadcasting cases in the samples. It just uses indexing to make the smaller tensors.